### ISSUE TITLE
Does not seem to work with source mapping

### ISSUE THREAD

**Issue by scottburch** [i-like-robots/rewireify#12]
Thanks for porting this to browserify.

browserify -t [reactify --es6 --target es5] -t rewireify spec/$FILE -o build/$FILE --debug

Adding the --debug flag with rewireify throws parsing errors.  I would love to have source maps also.

**Comment by i-like-robots** [comment]
Thanks for reporting... I confess I've never attempted to use this with the `--debug` flag so I'll check it out.

**Comment by scottburch** [comment]
Weird, I swapped reactify and wireify and it seems to work.

browserify -t rewireify -t [reactify ...] ...

**Comment by i-like-robots** [comment]
Just to confirm I recreate this and *fix* it as you describe. I'll investigate further.

**Comment by srosato** [comment]
Any news on this? I am trying this with tsify and debowerify with typescript and I am getting a parser error too on the source maps.

    browserify -v -d -p [tsify --jsx=react] -t rewireify -t debowerify

I also tried:

    browserify -v -d -t rewireify -p [tsify --jsx=react] -t debowerify

**Comment by scottburch** [comment]
I know this probably does not help, but I thought I would mention it since I was in the discussion.  I switched from using browserify to webpack.  After the learning curve, I find it to be much more powerful and easier.

**Comment by srosato** [comment]
Yeah well I will probably look into it too. For now I can confirm I am able to make it work with reactify and rewireify by swapping the arguments like you previously said, but I get the parse error when using tsify for typescript and I've been digging most of the afternoon for now...
